Auth/class.LDAPAuthenticator.php

```diff
@@ -154,11 +160,11 @@ public function getAndBindServer($bindWrite = false)
         $server->connect($this->host);
         if($bindWrite === false)
         {
-            $ret = $server->bind();
+          $ret = $server->bind($this->bindRODistinguishedName, $this->bindROPassword);
```

Comment:
I would add a check here. If $this->bindRODistinguishedName === false then still call $ret->bind() with no params. That would allow for the current operating mode to still function as is right?